### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by LKajan**

### CODE REVIEW

File: src/qgis_venv_creator/create_qgis_venv.py [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+
+
+class Windows(MultiQgisPlatform):
+    @classmethod
+    def _find_qgis_installations(cls, custom_search_path_pattern: str | None = None) -> list[Path]:
+        """Find all QGIS installations from the Windows system."""
+
+        possible_qgis_installation_generators = [
+            Path("C:/Program Files").glob("QGIS*/apps/qgis*/"),
+            Path("C:/OSGeo4W/apps").glob("qgis*/"),
```
Comment (JuhoErvasti) [inline src/qgis_venv_creator/create_qgis_venv.py:286]: Possible installation path when using OSGeo installer is missing (OSGeo4W64).
```suggestion
            Path("C:/OSGeo4W/apps").glob("qgis*/"),
            Path("C:/OSGeo4W64/apps").glob("qgis*/"),
```

**Review by JuhoErvasti — COMMENTED** [review]
Tested on Ubuntu and Windows, venv creation itself works fine. However I think for Windows a possible installation path should be added; C:/OSGeo4W64

File: src/qgis_venv_creator/create_qgis_venv.py (hunk 1 above)
Comment (LKajan) [reply src/qgis_venv_creator/create_qgis_venv.py:287]: Thanks, Added that now. That's so old I even forgot that possibility.

**Review by nmaarnio — APPROVED** [review]
Tested on Ubuntu and works

**Review by JuhoErvasti — APPROVED** [review]

**Pull request merged**